src/stylesheets/components/_card.scss

```diff
@@ -44,13 +45,15 @@
     }
 
     &--left, &--center {
+      svg {
+        @include u-margin-right(1);
+      }
       > * {
         display: inline-block;
```

Comment:
Use uswds utitlities `@include u-display('inline-block')` everywhere, there are several of these throughout the file. Note: I personally do not see the utility of this but it is a rule.